project/source/Scripts/Objects/GelMoldSubsceneManager.gd

```diff
@@ -18,6 +18,10 @@ var contents = []
 var hasComb = false
 var combObject = null
 
+# This determines whether it can be filled,
+# so that it can prevent taking substances when another object needs that substance
+var Fillable: bool = true
```

Comment:
I agree that I lean towards the former where we have that general property of `active` or something along those lines.

If we opt for that change, I think it would be best to set this `active` property within `ObjectSlot.gd` within the `TryInteract()` and `_on_GelBoatSlot_input_event` functions.